sopel/loader.py

```diff
@@ -104,10 +104,35 @@ def clean_callable(func, config):
 
 
 def is_triggerable(obj):
-    return any(
-        hasattr(obj, attr)
-        for attr in ('rule', 'event', 'intents', 'commands',
-                     'nickname_commands'))
+    """Check if ``obj`` can handle the bot's triggers.
+
+    :param obj: any :term:`function` to check.
+    :return: ``True`` if ``obj`` can handle the bot's triggers.
+
+    A triggerable is a callable that will be used by the bot to handle a
+    particular trigger (i.e. an IRC message): it can be a regex rule, an event,
+    an intent, a command or nickname command. However, it must not be a job
```

Comment:
```suggestion
    an intent, a command, or a nickname command. However, it must not be a job
```

Note to self: Will require another rebase of #1660.